test/performance/odp_stash_perf.c

```diff
@@ -0,0 +1,483 @@
+	       "\n"
+	       "  -c, --num_cpu <num>    Number of worker threads. Default: 1\n"
+	       "  -n, --num_stash <num>  Number of stash. Default: 1\n"
+	       "  -b, --burst_size <num> Number of burst size. Default: 1\n"
+	       "  -s, --stash_size <num> Stash size. Default: 1000\n"
+	       "  -r, --num_round <num>  Number of rounds\n"
+	       "  -m, --strict <arg>     Strict size stash. Default: 0\n"
+	       "  -h, --help             This help\n"
+	       "\n");
+}
+
+static int parse_options(int argc, char *argv[], test_options_t *test_options)
+{
+	int opt;
+	int long_index;
+	int ret = 0;
+
+	static const struct option longopts[] = {
+		{"num_cpu",    required_argument, NULL, 'c'},
+		{"num_stash",  required_argument, NULL, 'n'},
+		{"burst_size", required_argument, NULL, 'b'},
+		{"stash_size", required_argument, NULL, 's'},
+		{"num_round",  required_argument, NULL, 'r'},
+		{"strict",     no_argument,       NULL, 'm'},
+		{"help",       no_argument,       NULL, 'h'},
+		{NULL, 0, NULL, 0}
+	};
+
+	static const char *shortopts = "+c:n:b:s:r:mh";
+
+	test_options->num_cpu    = 1;
+	test_options->num_stash  = 1;
+	test_options->max_burst  = 1;
+	test_options->stash_size = 1000;
+	test_options->num_round  = 1000;
+	test_options->strict     = 0;
+
+	while (1) {
+		opt = getopt_long(argc, argv, shortopts, longopts, &long_index);
+
+		if (opt == -1)
+			break;
+
+		switch (opt) {
+		case 'c':
+			test_options->num_cpu = atoi(optarg);
+			break;
+		case 'n':
+			test_options->num_stash = atoi(optarg);
+			break;
+		case 'b':
+			test_options->max_burst = atoi(optarg);
+			break;
+		case 's':
+			test_options->stash_size = atoi(optarg);
+			break;
+		case 'r':
+			test_options->num_round = atoi(optarg);
+			break;
+		case 'm':
+			test_options->strict = 1;
+			break;
+		case 'h':
+			/* fall through */
+		default:
+			print_usage();
+			ret = -1;
+			break;
+		}
+	}
+
+	if (test_options->num_stash > MAX_STASHS) {
+		ODPH_ERR("Too many stashes %u. Test maximum %u.\n",
+			 test_options->num_stash, MAX_STASHS);
+		return -1;
+	}
+
+	return ret;
+}
+
+static int create_stashes(test_global_t *global)
+{
+	uint32_t i;
+	uint32_t tmp = 0;
+	test_options_t *test_options = &global->options;
+
+	uint32_t num_stash = test_options->num_stash;
+	uint32_t num_round = test_options->num_round;
+	odp_stash_t *stash = global->stash;
+	odp_stash_capability_t stash_capa;
+
+	printf("\nTesting %s stashs\n",
+	       test_options->strict == 0 ? "NORMAL" : "STRICT_SIZE");
+	printf("  num rounds           %u\n", num_round);
+	printf("  num stashs           %u\n", num_stash);
+	printf("  stash size           %u\n", test_options->stash_size);
+	printf("  max burst size       %u\n", test_options->max_burst);
+
+	if (odp_stash_capability(&stash_capa, ODP_STASH_TYPE_DEFAULT)) {
+		printf("stash capability failed\n");
+		return -1;
+	}
+
+	if (test_options->stash_size > stash_capa.max_num_obj) {
+		ODPH_ERR("Max stash size supported %lu\n", stash_capa.max_num_obj);
```

Comment:
Should use `%" PRIu64 "` to fix the CI failures.

Btw. if you haven't noticed, you can run the x86 CI also in your forked repo after enabling it in settings:
`Settings -> Actions -> General -> Allow all actions and reusable workflows`

Reply:
Sorry missed the x86 Testing. updated.